<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddOrChangeRoom</class>
 <widget class="QDialog" name="AddOrChangeRoom">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>405</width>
    <height>359</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>房间信息</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background:url(:/new/prefix1/bg.jpg);</string>
  </property>
  <widget class="QLabel" name="title">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>10</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>50</y>
     <width>251</width>
     <height>251</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image:url();
QLineEdit{
background:#03a9f4;

color: rgb(255, 255, 255);
};</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLineEdit" name="roomId">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>10</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius:15px;
background:#03a9f4;

color: rgb(255, 255, 255);</string>
    </property>
    <property name="placeholderText">
     <string>请输入房间号</string>
    </property>
   </widget>
   <widget class="QLabel" name="roomIdName">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>20</y>
      <width>54</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>房间号：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>60</y>
      <width>54</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>折扣：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="discount">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>50</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius:15px;
background:#03a9f4;

color: rgb(255, 255, 255);</string>
    </property>
    <property name="placeholderText">
     <string>不填默认为1</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>100</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>价格：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="roomCost">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>90</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius:15px;
background:#03a9f4;

color: rgb(255, 255, 255);</string>
    </property>
    <property name="placeholderText">
     <string>请输入房间价格</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="roomType">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>130</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius:15px;
background:#03a9f4;

color: rgb(255, 255, 255);</string>
    </property>
    <property name="placeholderText">
     <string>请填入房间类型</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>140</y>
      <width>54</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>房间类型：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>180</y>
      <width>54</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>房间状态：</string>
    </property>
   </widget>
   <widget class="QComboBox" name="status">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>170</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius:15px;
background:#03a9f4;

color: rgb(255, 255, 255);</string>
    </property>
    <item>
     <property name="text">
      <string>可入住</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>不可入住</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>维修中</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>220</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>入住客户Id：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="guestsId">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>210</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius:15px;
background:#03a9f4;

color: rgb(255, 255, 255);</string>
    </property>
    <property name="placeholderText">
     <string>可不填写</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="submit">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>310</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
background:#03a9f4;
color:#fff;
border-radius:15px;
};
</string>
   </property>
   <property name="text">
    <string>提交</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
